### PR TITLE
DiscussionDetail component with Tests

### LINKED ISSUE #14: Create DiscussionDetail component
This component should receive a Discussion object and display all its information: Author, title, date and content.

### PR DESCRIPTION
**Pull request by diegotc86**
# DiscussionDetail component:
Close #14 
## Props
- `discussion`: Discussion object

## Functions
None

## Result
The component return a box with all properties of the discussion object:
image

## Tests
- DiscussionDetail component (only snapshot)
- Show correct attributes

### CODE REVIEW

File: src/components/discussion-detail.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      </header>
+      <div
+        css={{
+          fontSize: "1.1em",
+          fontWeight: "bold",
+          marginBottom: "0.25em",
+          marginTop: "0.5em"
+        }}
+      >
+        <span aria-label="Title">{discussion.title}</span>
```
Comment (byteconde) [inline src/components/discussion-detail.js]: Well done, this will help our users to know the main topic

File: src/index.js [hunk 2]
```diff
@@ -1,25 +1,51 @@
...
-ReactDOM.render(<Reaction />, $root);
+// ReactDOM.render(<Login />, document.getElementById("root"));
+// ReactDOM.render(<App />, document.getElementById("root"));
+ReactDOM.render(
+  <>
+    <Global
+      styles={{
+        body: {
+          background: "#2c3e50",
+          fontFamily: "Roboto, 'Open Sans', 'Helvetica Neue', sans-serif",
```
Comment (byteconde) [inline src/index.js]: Usually we should define more than one font family. Some coders from the community say that 2-3 fonts work great. You can check more info about this [click-here](https://www.quora.com/How-many-different-fonts-should-you-use-in-your-website)

File: src/index.js [hunk 3]
```diff
@@ -1,25 +1,51 @@
...
-const $root = document.getElementById("root");
-ReactDOM.render(<Reaction />, $root);
+// ReactDOM.render(<Login />, document.getElementById("root"));
+// ReactDOM.render(<App />, document.getElementById("root"));
+ReactDOM.render(
+  <>
+    <Global
+      styles={{
+        body: {
+          background: "#2c3e50",
```
Comment (byteconde) [inline src/index.js]: I think we should not define a global style that wraps a big 'feature' like the background. I mean, how about if we want to change the style of one of the views? We will have to overwrite the styles.

Comment (diegotc86) [reply src/index.js]: Lets see if this property give us problems in the next features.

**Pull request merged**